src/erlcloud_cur.erl

```diff
@@ -0,0 +1,133 @@
+-module(erlcloud_cur).
```

Comment:
although the docs for Cost&Usage Reports are placed together with budget API: https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/api-reference.html

in aws cli those are separate entities:

```
aws cur help
aws budgets help
```

So I decided to keep it in separate module, to have names compatible with `aws cli`